docs/docs/responses.mdx

```diff
@@ -258,3 +258,31 @@ responses:
   - text: "Hey, {name}. How are you?"
   - text: "Hey, {name}. How is your day going?"
 ```
+
+## Responses for Retrieval Intents
+
+If you are using retrieval intents in your assistant, you also need to add response templates
+for your assistant's replies to these intents in a separate file:
```

Comment:
let's remove the "separate file" part, @degiz has already created a PR to remove that restriction

Reply:
and even merged it! 🚀